qhub/template/image/scripts/install-code-server.sh

```diff
@@ -15,7 +15,7 @@ if [[ ! $(sha256sum install.sh) == "${expected_sum}  install.sh" ]];then
 fi
 
 mkdir /opt/tmpdir
-sh ./install.sh --method standalone --prefix /opt/tmpdir
+sh ./install.sh --method standalone --prefix /opt/tmpdir --version 4.5.0
```

Comment:
Thanks for working on this, this is written at two places, here and in line 20, please define a variable and use that at both places.

Reply:
Fixed it now.